### PR TITLE
1.14 support

### PR DESCRIPTION
**Pull request by gerricchaplin**

### CODE REVIEW

**Comment by gerricchaplin** [comment]
https://v1-14.docs.kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/ upscale delay no longer supported. Downscale delay switched to downscale-stabilization.

Corrected key from horizontalPodAutoscalerSyncPerdiod to horizontalPodAutoscalerSyncPeriod

File: aws/cluster/variables.tf [hunk 1]
```diff
@@ -205,20 +203,13 @@ variable "hpa-sync-period" {
   default = "30s"
 }
 
-variable "hpa-scale-down-delay" {
+variable "hpa-scale-downscale-stabilization" {
   type        = "string"
-  description = "After a downscale, wait at least for this duration before the next downscale"
+  description = "After an downscale, wait at least for this duration before the next downscale"
```
Comment (r2ronoha) [inline aws/cluster/variables.tf:208]: I wouldn't have changed that :)

File: aws/cluster/templates/cluster-spec.yaml [hunk 2]
```diff
@@ -33,6 +33,7 @@ ${etcd-clusters}
     - DefaultTolerationSeconds
     - NodeRestriction
     - Priority
+    - DenyEscalatingExec
```
Comment (r2ronoha) [inline aws/cluster/templates/cluster-spec.yaml:36]: This seems to be [deprecated](https://github.com/dataiku/dku-kubernetes/blob/master/CHANGELOG-1.14.md)?

Comment (gerricchaplin) [reply aws/cluster/templates/cluster-spec.yaml:36]: Yes, deprecated but not gone. Still need to do the work on its replacement so it stays for now.

File: aws/cluster/variables.tf (hunk 1 above)
Comment (gerricchaplin) [reply aws/cluster/variables.tf:208]: I'll change it next major merge. 😁

**Pull request merged**